submissions/2024-10-11-MLPepper-RECAP-Optimized-Fragments-Add-Iodines-v1.0/create_dataset.py

```diff
@@ -0,0 +1,140 @@
+from openff.qcsubmit.datasets import load_dataset, BasicDataset
+from openff.qcsubmit.common_structures import DDXSettings, SCFProperties, Metadata
+import tqdm
+import numpy as np
+
+
+def main():
+    # load the datasets optimised at aimnet2
+    esp_50k_I = load_dataset('dataset.json.bz2')
```

Comment:
This file isn't included since it has the same name as the final dataset--could you save it with a different name and upload it?